pkg/webhook/policy.go

```diff
@@ -307,6 +312,17 @@ func (h *validationHandler) validateConstraint(ctx context.Context, req admissio
 	return false, nil
 }
 
+func (h *validationHandler) validateConfigResource(ctx context.Context, req admission.Request) (bool, error) {
```

Comment:
We should just return one value, `error`. Error will be `nil` if the resource is valid.

Reply:
Done